_posts/2020-11-16-changelog-135.markdown

```diff
@@ -0,0 +1,118 @@
+    * Detect [E0364](https://doc.rust-lang.org/error-index.html#E0364) and [E0365](https://doc.rust-lang.org/error-index.html#E0365) compiler errors and suggest quick-fixes (by [@Kobzol])
+
+  <!-- https://github.com/intellij-rust/intellij-rust/pull/6304 -->
+    * Detect [E0449](https://doc.rust-lang.org/error-index.html#E0449) for enum variants and suggest quick-fix
+
+## Performance Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6368 -->
+* Introduce new macro expansion cache shared between different projects.
+It brings slightly speedup if you open several projects that use the same library.
+Note, you can turn it off via `Registry... | org.rust.lang.macros.persistentCache` option
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6369 -->
+* Allow [language injections](https://www.jetbrains.com/help/clion/using-language-injections.html#cancel-injections) in string literals inside macro calls
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6326 -->
+* Take execution context into account in the [Run Anything](https://www.jetbrains.com/help/clion/rust-support.html#cargo-support) popup.
+  Now you can specify an execution context by holding down `Alt` key or by manually choosing a directory
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6238 -->
+* Fix false positive `Match must be exhaustive` error in case of uninhabited types, e.g. empty enums (by [@fan-tom])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6207 -->
+* Do not show `Add wildcard pattern` intention for empty match expressions
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6356 -->
+* Update GDB pretty-printers to render `BTreeMap`/`BTreeSet` on Rust `1.44.0` or higher
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6314 -->
+* Provide proper rustc version by `Create New Issue` action which allows opening issue in our [tracker](https://github.com/intellij-rust/intellij-rust/issues/new) right from your IDE
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6272 -->
+* Do not load Cargo project structure twice. It makes Cargo projects loading a bit faster
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6313 -->
+* Do not suggest lints in paths completion if path starts with `::` (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6298 -->
+* Fix false positive `Unused variable` warning in Rust code containing `async` blocks
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6341 -->
+* Provide quick documentation for primitive types in stdlib
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6291 -->
+* Allow duplicate parameter bindings for trait methods without a body ([E0415](https://doc.rust-lang.org/error-index.html#E0415) compiler error) (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6285 -->
+* Improve the parser recovering in case of incomplete paths
+
+## Internal Improvements
```

Comment:
Looks like https://github.com/intellij-rust/intellij-rust/pull/6358 is missin here
Interesting fact that it's the first PR with special changelog description but this branch doesn't contain the corresponding changes in `changlog.py`

Reply:
Oh, I think I forgot to regenerate the changelog after this PR was merged